ext/fast_polylines/fast_polylines.c

```diff
@@ -24,6 +24,18 @@
 #define rdbg(...)
 #endif
 
+typedef unsigned int uint;
+
+// Speed up the code a bit by not calling pow(10.0, precision) since it's always 10^precision
+// Convert that into a lookup instead of doing the calculation
+const double PRECISION_VALUES[MAX_PRECISION+2] = {1.0, 10.0, 1.0e+2, 1.0e+3, 1.0e+4, 1.0e+5, 1.0e+6, 1.0e+7, 1.0e+8, 1.0e+9, 1.0e+10, 1.0e+11, 1.0e+12, 1.0e+13, 1.0e+14};
+static inline double fast_pow10(uint);
```

Comment:
I guess we could get rid of that line since implementation is a one liner!

Reply:
Yes, of course. I wasn't sure what you follow at Klaxit and did not want to impose!